ClemBot.Bot/bot/cogs/search_cog.py

```diff
@@ -0,0 +1,159 @@
+import logging
+import re
+import json
+import aiohttp
+import discord
+import discord.ext.commands as commands
+import bot.extensions as ext
+from enum import Enum
+from markdownify import markdownify
+from urllib.parse import urlencode
+from urllib.parse import unquote_plus
+from bot.consts import Colors
+from bot.messaging.events import Events
+
+log = logging.getLogger(__name__)
+max_related_topics = 5
+image_url = 'https://duckduckgo.com'
+search_url = 'https://api.duckduckgo.com/'
+icon_url = 'https://i.imgur.com/Knq2kVa.png'
+related_topics_pattern = re.compile('(https://duckduckgo.com/)([acdne]/)?(\\S+)', re.IGNORECASE)
```

Comment:
Module Level constants are SCREAMING_SNAKE_CASE with the exception of the logger for legacy reasons